indexing-service/src/test/java/io/druid/indexing/overlord/http/OverlordResourceTest.java

```diff
@@ -197,8 +200,27 @@ public void testSecuredGetCompleteTasks()
   {
     expectAuthorizationTokenCheck();
 
-    List<String> tasksIds = ImmutableList.of("id_1", "id_2", "id_3");
-    EasyMock.expect(taskStorageQueryAdapter.getRecentlyFinishedTaskStatuses(null)).andReturn(
+    List<String> tasksIds = ImmutableList.of("id_1", "id_2");
```

Comment:
why eliminate the 3rd task id?

Reply:
There was no specific reason. Reverted the same.